Revit_oM/Misc/RevitGeometry.cs

```diff
@@ -20,50 +20,43 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
-using System.ComponentModel;
-
 using BH.oM.Base;
-using BH.oM.Reflection.Attributes;
+using BH.oM.Geometry;
+using System.Collections.Generic;
+using System.ComponentModel;
 
-namespace BH.Engine.Adapters.Revit
+namespace BH.oM.Adapters.Revit
 {
-    public static partial class Query
+    [Description("Fragment containing the geometry extracted from Revit element represented by the BHoM object.")]
+    public class RevitGeometry : IFragment, IImmutable
     {
         /***************************************************/
-        /****              Public methods               ****/
+        /****             Public Properties             ****/
         /***************************************************/
 
-        [Deprecated("3.2", "BH.Engine.Adapters.Revit.Query.SpaceId is not used any more.")]
-        [Description("Gets integer representation of ElementId of Revit space element correspondent to given BHoMObject. This value is stored in CustomData under key SpaceID.")]
-        [Input("bHoMObject", "BHoMObject to be queried.")]
-        [Output("elementId")]
-        public static int SpaceId(this IBHoMObject bHoMObject)
-        {
-            if (bHoMObject == null)
-                return -1;
+        [Description("Edge curves of Revit element represented by the BHoM object carrying this fragment.")]
+        public virtual List<ICurve> Edges { get; set; } = null;
```

Comment:
If you intend this to be immutable, would change the following:

At least this change:
```suggestion
        public virtual List<ICurve> Edges { get;  } = null;
```
But even more protected, could do this. (This requires the additional using of `using System.Collections.ObjectModel;`):

```suggestion
        public virtual ReadOnlyCollection<ICurve> Edges { get;  } = null;
```

Reply:
Well spotted, all fixed!